### PR TITLE
@W-16217108@ Update package_version_create.md

### PR DESCRIPTION
**Pull request by penelopy**
Edited the Help text that displays with packageVersionCreatePerformingValidations and edited the definition-file flag

### CODE REVIEW

**Comment by penelopy** [comment]
@mradulsf Here is the PR with the Help text changes. Here is the WI I created. I'll ping Rob re this WI https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001wQOe3YAG/view

File: messages/package_version_create.md [hunk 1]
```diff
@@ -214,8 +214,8 @@ Version create.
 # packageVersionCreatePerformingValidations
 
 The validations for this package version are in progress, but you can now begin testing this package version.
-To determine whether all package validations completed successfully, run sf package version create report and review the Async Validation Status.
-Async validated package versions can be promoted only if all validations completed successfully.
+To determine whether all package validations complete successfully, run sf package version create report --package-create-request-id  08cxx and review the Status.
```
Comment (jshackell-sfdc) [inline messages/package_version_create.md:217]:
```suggestion
To determine whether all package validations complete successfully, run "sf package version create report --package-create-request-id  08cxx" and review the Status.
```

Comment (jshackell-sfdc) [reply messages/package_version_create.md:217]: Just a suggestion, to make it easier to see what the actual command is.

File: messages/package_version_create.md [hunk 2]
```diff
@@ -47,7 +47,7 @@ Path to the directory that contains the contents of the package.
 
 # flags.definition-file.summary
 
-Path to a definition file similar to scratch org definition file that contains the list of features and org preferences that the metadata of the package version depends on.
+Path to a definition file similar to scratch org definition file that contains the list of features and org preferences that the metadata of the package version depends on. For a patch version, the features specified in this file are ignored, and instead the features specified for the ancestor version are used.
```
Comment (jshackell-sfdc) [inline messages/package_version_create.md:50]: FYI that the summary is pretty long now and goes into two lines in the --help:

image

That's fine, if you're okay with it!  Just wanted to point it out.  

If you want to keep the summary short, you can always put this new text in a `flags.definition-file.description.`

**Pull request closed without merging**